src/consensus/vote.cpp

```diff
@@ -543,51 +543,48 @@ bool VoteManager::voteValidation(std::shared_ptr<Vote>& vote, size_t dpos_total_
 bool VoteManager::pbftBlockHasEnoughValidCertVotes(SyncBlock& pbft_block_and_votes, size_t dpos_total_votes_count,
                                                    size_t sortition_threshold, size_t pbft_2t_plus_1) const {
   if (pbft_block_and_votes.cert_votes.empty()) {
-    LOG(log_er_) << "No any cert votes! The synced PBFT block comes from a "
-                    "malicious player.";
+    LOG(log_er_) << "No any cert votes! The synced PBFT block comes from a malicious player.";
```

Comment:
Should be "No cert votes provided.  Synced PBFT block comes from a malicious player."

Reply:
Fixed log message, thanks